src/TwigComponent/src/ComponentAttributes.php

```diff
@@ -41,7 +43,11 @@ public function __toString(): string
                 fn (string $key) => !isset($this->rendered[$key])
             ),
             function (string $carry, string $key) {
-                if (preg_match(self::NESTED_REGEX, $key)) {
+                if (
+                    preg_match(self::NESTED_REGEX, $key)
+                    && !preg_match(self::ALPINE_REGEX, $key)
+                    && !preg_match(self::VUE_REGEX, $key)
+                ) {
                     return $carry;
                 }
```

Comment:
`preg_match` calls can be expensive, and can be avoid by quickly checking if the attribute name contains a `:`. If it's not the case we early exit (without calling `preg_match`), otherwise we continue.

```suggestion
                if (
                    str_contains($key, ':')
                    && preg_match(self::NESTED_REGEX, $key)
                    && !preg_match(self::ALPINE_REGEX, $key)
                    && !preg_match(self::VUE_REGEX, $key)
                ) {
                    return $carry;
                }
```

Can you add the same check to `\Symfony\UX\TwigComponent\ComponentAttributes::nested` please? 

Thanks!

PS: Not related to your PR, but I'm not a fan about all the `array_reduce`/`array_filter`/`array_keys` calls where a `foreach` could be enough. We can open a new PR to rewrite and optimize that :)

Reply:
> Can you add the same check to \Symfony\UX\TwigComponent\ComponentAttributes::nested please?

The str_contains will be enough here, as if someone want to explicitely ask for .nested("x-on") let's not block this.

> I'm not a fan about all the array_reduce/array_filter/array_keys calls where a foreach could be enough. We can open a new PR to rewrite and optimize that :)

Yes, will do leter

Reply:
Done here in the end, i'll let you check